packages/ccd-js-gen/README.md

```diff
@@ -0,0 +1,484 @@
+# Smart Contract Client Generator <!-- omit in toc -->
+
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.0-4baaaa.svg)](https://github.com/Concordium/.github/blob/main/.github/CODE_OF_CONDUCT.md)
+
+Generate TypeScript/JavaScript code for interating with smart contracts and modules on the Concordium blockchain.
+
+- Functions for dry-running and calling entrypoints of the smart contract.
+- Functions for constructing parameters.
+- Parsing logged events, return values and error messages.
+- Structured types for parameter, logged events, return values and error messages.
+
+The code is generated from deployable smart contract modules, meaning it can be done with any smart contract available locally and any smart contract deployed on chain.
+
+### Example usage of a generated client
+
```

Comment:
Please state here what contract this is generated from, maybe show the entrypoint that you are invoking. Otherwise it's quite hard to understand the transfer example.

Reply:
I have added more to this now